LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2022 Povio Inc. <services@povio.com>
+Copyright (c) 2023 Povio Inc. <services@povio.com>
```

Comment:
Hm, did you branch off #234 ? A lot of changes related to copyright in this PR

Reply:
Wrong rebase yes. It's fixed now.